nixos/modules/misc/ids.nix

```diff
@@ -414,6 +415,7 @@
       btsync = 113;
       #minecraft = 114; # unused
       #monetdb = 115; # unused (not packaged), removed 2016-09-19
+      vault = 115;
```

Comment:
It looks like the service will change permission of all files related to this service on start. It might be not necessary to allocate static uids/gids.